source/pic2card/docker/Dockerfile

```diff
@@ -11,10 +11,10 @@
 # usecase at hand.
 #
 # To build docker image with TFS support
-#   $docker build -t imaginea.azurecr.io/imaginea/api_frozen_mystique:v0.6 .
+#   $ docker build -t imaginea.azurecr.io/imaginea/api_frozen_mystique:v0.6 .
```

Comment:
Check the docker path is still valid?

Reply:
it's commented out, part. Seems the main branch change caused some more conflicts, due to some .github folder removals. Will address those via separate PR or updating this one.